tools/autograd/load_derivatives.py

```diff
@@ -88,6 +91,9 @@ def create_derivative(f: NativeFunction, formula: str, var_names: Tuple[str, ...
     formula, saved_inputs = saved_variables(formula, arguments, var_names)
     formula, saved_outputs = saved_variables(formula, named_returns, var_names)
 
+    named_gradients = {name for name in available_named_gradients
```

Comment:
nit: used_named_gradients ?

Reply:
done. do you want me to change the field name on Derivative as well?